examples/flux_operator_ca_hpa/operator-minicluster/hpc7g-configs/flux-operator-arm.yaml

```diff
@@ -0,0 +1,1412 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
Are you running the full experiments this weekend? I figure you can remove the arm config if we decide we don't need the hpc7g.